source/postprocess/command.cc

```diff
@@ -29,7 +29,7 @@ namespace aspect
     Command<dim>::execute (TableHandler &)
     {
       if (on_all_processes ||
-          (Utilities::MPI::this_mpi_process(this->get_mpi_communicator()) == 0))
```

Comment:
Are these dealii:: prefixes here and in the other instances still necessary? I though the `using` statements had fixed that?